CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Welcome!
+Thank you for your interest in Dashing! This exceptionally handsome
+framework welcomes all input, but being stylish requires certain protocols be followed. :bowtie:
+
+Below you will find a set of guidelines that will ensure the best outcome for you _and_ Dashing.
+<a name="issues"></a>
+## Have an Issue
+If you run into problems with Dashing, please take these steps before submitting an issue:
+
+1. Check the [Troubleshooting Guide]() in the wiki.
```

Comment:
I'd just link them here for now: https://github.com/Shopify/dashing/wiki#how-tos